### ISSUE TITLE
Failed to Import Settings

### ISSUE THREAD

**Issue by AlifianK** [TuxedoTako/4chan-xt#16]
Version 2.3.1
Console log:
```
userscript.html?name=4chan-XT.user.js&id=9fb8236b-e1ae-4147-a386-1848c7cbb1ac:15479 [Violation] 'change' handler took 1367ms
userscript.html?name=4chan-XT.user.js&id=9fb8236b-e1ae-4147-a386-1848c7cbb1ac:15505  TypeError: Cannot read properties of undefined (reading 'replace')
    at Object.loadletter (userscript.html?name=4chan-XT.user.js&id=9fb8236b-e1ae-4147-a386-1848c7cbb1ac:15607:55)
    at Object.loadSettings (userscript.html?name=4chan-XT.user.js&id=9fb8236b-e1ae-4147-a386-1848c7cbb1ac:15955:43)
    at reader.onload (userscript.html?name=4chan-XT.user.js&id=9fb8236b-e1ae-4147-a386-1848c7cbb1ac:15493:35)
reader.onload @ userscript.html?name=4chan-XT.user.js&id=9fb8236b-e1ae-4147-a386-1848c7cbb1ac:15505
load (async)
onImport @ userscript.html?name=4chan-XT.user.js&id=9fb8236b-e1ae-4147-a386-1848c7cbb1ac:15491
```

**Comment by TuxedoTako** [comment]
I see. I removed the "XT " prefix in the version number, because the chrome extension gave an error with a non numeric version, and I don't know how userscripts handle it, but there is some code for a different fork that also went with 2.x.x: https://github.com/TuxedoTako/4chan-xt/blob/6bcf255cbd287b9a02dfabf0c7fa833ca3525e42/src/General/Settings.tsx#L823-L833

I guess this is also a problem when moving from XT back to X, but I can't change anything in that repo.

**Comment by TuxedoTako** [comment]
Should be fixed in [2.3.2](https://github.com/TuxedoTako/4chan-xt/releases). I also opened https://github.com/ccd0/4chan-x/pull/3429 so this is fixed in upstream 4chan X, if that one gets updated again.

**Comment by AlifianK** [comment]
Thanks for the quick fix!
